cluster/saltbase/salt/fluentd-es/fluentd-es.manifest

```diff
@@ -11,7 +14,7 @@ containers:
         mountPath: /var/lib/docker/containers
       - name: varlog
         mountPath: /varlog
-volumes:
+  volumes:
```

Comment:
Did you try kubectl create --validate?

Please read v1beta3 conversion tips:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api.md#v1beta3-conversion-tips

volumes could be failing validation. source should be inlined and hostDir has been replaced by hostPath:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L203

Also, please rename the file to .yaml.